File: 2023_11_03/abc049b.py
H, W = map(int, input().split())
Chw = []

for i in range(0, H):
    Chw.append(list(input()))

print(Chw)

for i in range(1, H * 2 + 1):
    str = ""

    for j in range(1, W + 1):
        str += Chw[(i + 2) // 2][j + 1]

    print(str)

File: 02/008/2023_12_01/abc049b.jl
H, W = parse.(Int, split(readline()))
Chw = []

for i in 1:H
    push!(Chw, split(readline(), ""))
end

for i in 1:(H*2)
    for j in 1:W
        print(Chw[(i+1)÷2][j])
    end

    println("")
end
